packages/shared/lib/core/network/actions/getAndUpdateNodeInfo.ts

```diff
@@ -0,0 +1,15 @@
+import { profileManager } from '@lib/wallet'
+import { get } from 'svelte/store'
+import { INodeInfoResponse } from '../interfaces'
+import { updateNodeInfo } from '../stores/node-info.store'
+
+export async function getAndUpdateNodeInfo(): Promise<void> {
+    let nodeInfoResponse: INodeInfoResponse | null
```

Comment:
We don't have to explicitely type `null` or `undefined`

Reply:
Done